dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -37,9 +37,18 @@
 module Pd::Application
   class Teacher1819Application < TeacherApplicationBase
     include ::Pd::Teacher1819ApplicationConstants
+    include RegionalPartnerTeacherconMapping
 
     validates_uniqueness_of :user_id
 
+    has_one :pd_teachercon1819_registration,
+      class_name: 'Pd::Teachercon1819Registration',
+      foreign_key: 'pd_application_id'
+
+    serialized_attrs %w(
+      auto_assigned_enrollment_id
+    )
+
     # @override
```

Comment:
I assume that this code will never get executed in real life again because it's 1819? If it is dead code, should we remove it completely instead of updating it? (I guess "removing" could be deleting the route configs so request won't hit this code path.)

Reply:
Since instances of these models still exist in our database, and at some point we plan to allow partners to download CSVs of their past applications, I want to preserve the data, including the [associations](https://guides.rubyonrails.org/association_basics.html) between these models.

Currently, no routes are loading these applications.